brokerpaks/aws-ses/terraform/provision/main.tf

```diff
@@ -92,7 +92,10 @@ locals {
 resource "aws_sesv2_email_identity" "identity" {
   configuration_set_name = aws_sesv2_configuration_set.config.configuration_set_name
   email_identity         = local.domain
-  # Should match https://github.com/cloud-gov/go-broker-tags/blob/main/tags.go#L10
+
+  tags = {
+    "new field" = var.new_field
```

Comment:
Do we actually want a tag named "new field"

Reply:
Resolved in next commit 👍